### PR TITLE
chore: return the kernel version when ping api

### LINKED ISSUE #279: the ping api needs to return the kernel version
### Describe the bug（Bug 描述）

expected 

```bash
✗ curl -i -GET 'http://127.0.0.1:8086/ping'
HTTP/1.1 204 No Content
Connection: keep-alive
Content-Type: application/json
Date: Wed, 24 May 2023 02:40:15 GMT
Keep-Alive: timeout=4
Proxy-Connection: keep-alive
Request-Id: 502649ca-f9dc-11ed-800a-acde48001122
X-Influxdb-Build:  #🌟 🌟 
X-Influxdb-Version:  #🌟 🌟 
X-Request-Id: 502649ca-f9dc-11ed-800a-acde48001122
```

### To Reproduce（Bug 复现步骤）

1. start openGemini service.
2. `curl -i -GET 'http://127.0.0.1:8086/ping'`

### Expected behavior（期望结果）

```bash
✗ curl -i -GET 'http://127.0.0.1:8086/ping'
HTTP/1.1 204 No Content
Connection: keep-alive
Content-Type: application/json
Date: Wed, 24 May 2023 02:40:15 GMT
Keep-Alive: timeout=4
Proxy-Connection: keep-alive
Request-Id: 502649ca-f9dc-11ed-800a-acde48001122
X-Geminidb-Build:  OSS    #🌟 🌟 
X-Geminidb-Version: v1.0.2   #🌟 🌟 
X-Request-Id: 502649ca-f9dc-11ed-800a-acde48001122
```

### Screenshots（屏幕截图）

_No response_

### Logs（完整的错误日志）

_No response_

### Additional context（其他的一些补充内容）

_No response_

### PR DESCRIPTION
**Pull request by xmh1011**
### What problem does this PR solve?

Issue Number: fix #279 

### What is changed and how it works?
Please describe how it works
```
curl -i -GET 'http://127.0.0.1:8086/ping'
HTTP/1.1 204 No Content
Content-Type: application/json
Request-Id: af2679b8-f9e1-11ed-8001-b6993de84a29
X-Geminidb-Build: OSS
X-Geminidb-Version: v1.0.2
X-Request-Id: af2679b8-f9e1-11ed-8001-b6993de84a29
Date: Wed, 24 May 2023 03:18:41 GMT
```

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B
- [ ] Test cases to be added
- [ ] No code

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: open_src/influx/httpd/handler.go [hunk 1]
```diff
@@ -109,6 +109,12 @@ type Route struct {
 	HandlerFunc    interface{}
 }
 
+const (
+	// change the default value when a new version released
+	version = "v1.0.2"
```
Comment (shilinlee) [inline open_src/influx/httpd/handler.go:114]: Maybe use 

https://github.com/openGemini/openGemini/blob/df125d9175ec98695eb4a8de8fe19e1e7677641c/app/ts-sql/main.go#L37

**Review by shilinlee — CHANGES_REQUESTED** [review]

File: app/ts-sql/sql/server.go [hunk 2]
```diff
@@ -115,6 +115,8 @@ func NewServer(conf config.Config, cmd *cobra.Command, logger *Logger.Logger) (a
 		metaUseTLS:    false,
 		config:        c,
 	}
+	s.httpService.Handler.Version = cmd.Version
+	s.httpService.Handler.BuildType = "OSS"
```
Comment (scuzyj) [inline app/ts-sql/sql/server.go:119]: what is the meaning of "OSS"?

File: open_src/influx/httpd/handler.go [hunk 3]
```diff
@@ -351,9 +351,9 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	defer atomic.AddInt64(&statistics.HandlerStat.ActiveRequests, -1)
 	start := time.Now()
 
-	// Add version and build header to all InfluxDB requests.
-	w.Header().Add("X-Influxdb-Version", h.Version)
-	w.Header().Add("X-Influxdb-Build", h.BuildType)
+	// Add version and build header to all openGemini requests.
```
Comment (scuzyj) [inline open_src/influx/httpd/handler.go:354]: The comments and codes must be consistent. Select either Geminidb or openGemini.

File: app/ts-sql/sql/server.go (hunk 2 above)
Comment (shilinlee) [reply app/ts-sql/sql/server.go:119]: **Object Storage Service**

**Review by shilinlee — APPROVED** [review]

**Pull request merged**